jax/__init__.py

```diff
@@ -40,7 +40,7 @@
 from ._src.config import (
   config, enable_checks, check_tracer_leaks, checking_leaks,
   debug_nans, debug_infs, log_compiles, default_matmul_precision,
-  numpy_rank_promotion
+  numpy_rank_promotion, hlo_metadata_source_file_prefix_removal_regex
```

Comment:
I don't think I'd export this in the jax namespace. It's not something users should consume.

Reply:
I'm not sure how else to access the context manager. We set jax.config to be the Config singleton object inside config.py. Is there a way to access the config module itself?

I can also add the try/finally logic to the test itself, it just seems silly given we have this nice context manager.

Reply:
`import jax._src.config` ?

Reply:
Oh, duh. I forgot we can do that. That works!